POLICIES.md

```diff
@@ -0,0 +1,56 @@
+# Track Policies
+
+This document:
+
+1. describes all current track-wide policies; and
+2. lists all policies that should be reviewed in response to any given track event.
+
+Maintainers should refer to this document each time they review a pull request.
+
+Our policies are not set-in-stone. They represent directions chosen at a point in time with the information/context available at that time. We actively encourage contributors to review and critique them. If you have read through the references that informed an existing policy and still feel it should be improved or removed, please comment and describe:
+
+1. your alternative perspective;
+2. your proposed changes; and
+3. how your proposal balances your own perspective with those already raised during prior discussions.
+
+## Policy Descriptions
```

Comment:
Does it make sense to sort the policies in the order they are mentioned in the table?  Might help make it more navigable.